cinema/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from cinema.views import movie_list, movie_detail
+
+urlpatterns = [
+    path("movies/", movie_list, name="movie-list"),
+    path("movies/<int:pk>/", movie_detail, name="movie")
```

Comment:
```suggestion
    path("movies/<int:pk>/", movie_detail, name="movie-detail")
```
`detail` would be better